ProviderFactory/AlgoliaPlacesFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the BazingaGeocoderBundle package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Bazinga\GeocoderBundle\ProviderFactory;
+
+use Geocoder\Provider\AlgoliaPlaces\AlgoliaPlaces;
+use Geocoder\Provider\Provider;
+use Http\Discovery\HttpClientDiscovery;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class AlgoliaPlacesFactory extends AbstractFactory
+{
+    protected static $dependencies = [
+        ['requiredClass' => AlgoliaPlaces::class, 'packageName' => 'geocoder-php/algolia-place-provider'],
+    ];
+
+    protected function getProvider(array $config): Provider
+    {
+        $httplug = $config['httplug_client'] ?: HttpClientDiscovery::find();
+
+        return new AlgoliaPlaces($httplug, $config['apiKey'], $config['appId']);
+    }
+
+    protected static function configureOptionResolver(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'httplug_client' => null,
+            'apiKey' => null,
+            'appId' => null,
+        ]);
+
+        $resolver->setAllowedTypes('httplug_client', ['object', 'null']);
+        $resolver->setAllowedTypes('apiKey', ['string']);
```

Comment:
What about using `api_key` and `app_id` instead to be consistent with the others factories?

Reply:
👍